release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-networking"]
 === Networking
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+==== Dynamic use of non-reserved CPUs for OVS
+
+With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs by default in performance-tuned clusters with a CPU manager policy set to `static`. 
+This dynamic use of available, non-reserved CPUs maximizes compute resources for OVS and avoids network latency for workloads during periods of high demand. 
```

Comment:
here instead of `avoids network latency` I'd say `minimizes network latency`. Other than this, it looks good to me!